mandatory/1-fix-functions.js

```diff
@@ -10,20 +10,18 @@
 
 */
 
-function getMood() {
-  let isHappy = true;
-
-  if (isHappy) {
+function getMood(isHappy) {
+  if (isHappy === true) {
```

Comment:
This works and is a good answer. 

It might be interesting now to explore ways to refactor and simply this function. 

`if (isHappy === true)`

If isHappy is the same as true, do you need to make this check? What does isHappy _evaluate_ to? (Replace the left hand side with the value of isHappy and read the code again. )